docs/compact_ranges.md

```diff
@@ -210,3 +210,37 @@ the load on the coordinator with the load on "leaf" workers.
 
 The described approach can be used to implement a scalable tamper-evident log
 based on Merkle trees.
+
+### Updatable Proofs
+
+Logs based on Merkle trees are rarely static - usually they grow in append-only
+fashion. Proofs built for one state of the tree get outdated with later states.
+If proofs are made using compact ranges, they can be updated by merging in the
+compact range between the previous and the next tree size.
+
+For example, an inclusion proof for range `[L, R)` in a tree of size `N` can be
+represented with compact ranges `[0, L)` and `[R, N)`, as described in a
```

Comment:
/in a [section]/in the section on [arbitrary inclusion proofs] above/"

Reply:
Done.